src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -1488,6 +1488,11 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
      *  ensuring that symbol is initialized (i.e. type is completed).
      */
     def info: Type = try {
+      /* Register the position from the symbol we are completing iff it comes
+       * from a source file, which means that this symbol forces unpickling.
+       * It's useful to report on stub errors originating from this symbol. */
+      if (pos.isDefined)
+        self.lastKnownPosition = pos
```

Comment:
This should be skipped if the symbol is already complete (which is the common case and must avoid any overhead).